bokeh/models/annotations/legends.py

```diff
@@ -234,6 +234,16 @@ def __init__(self, *args, **kwargs) -> None:
         (i.e. `low=0`), the tick and tick labels won't be rendered.
     """)
 
+    low_cutoff = Nullable(Float, help="""
+    The lowest value to show in the color bar. The whole of the color entry
+    containing this value is shown.
+    """)
+
+    high_cutoff = Nullable(Float, help="""
+    The highest value to show in the color bar. The whole of the color entry
+    containing this value is shown.
+    """)
```

Comment:
Would it make sense to use names like `display_low` and `display_high` (or similar) to help reinforce that these properties only affect the visual presentation, and not the color mapping?

Reply:
Yes, those are better names. I've also changed `low_index` and `high_index` to `index_low` and `index_high` to better associate them with `display_low` and `display_high`.